test/spec/modules/liveIntentIdSystem_spec.js

```diff
@@ -89,6 +89,9 @@ describe('LiveIntentId', function() {
       emailHash: '58131bc547fb87af94cebdaf3102321f'
     }});
     setTimeout(() => {
+      const packageJson = require('../../../package.json')
+      const version = packageJson.version
+      expect(server.requests[0].url).to.contain(`tv=${version}`)
```

Comment:
This test can be separated from the others. That way the test scopes will be clearer.